companion/src/firmwares/moduledata.cpp

```diff
@@ -247,12 +247,19 @@ QString ModuleData::subTypeToString(int type) const
     "915MHz"
   };
 
+  static const char * PpmSubTypeStrings[PPM_NUM_SUBTYPES] = {
+    "No Telemetry",
+    "MLink"
+  };
+
```

Comment:
lowercase 1st letter ppmSubType....
add translation hooks for strings
tr("No Telemetry")
tr("MLink")

Reply:
done